apis/extension/load_aware.go

```diff
@@ -27,6 +28,12 @@ const (
 	// AnnotationCustomUsageThresholds represents the user-defined resource utilization threshold.
 	// For specific value definitions, see CustomUsageThresholds
 	AnnotationCustomUsageThresholds = SchedulingDomainPrefix + "/usage-thresholds"
+	// AnnotationCustomForceEstimationSecondsAfterPodScheduled represents the user-defined
+	// force estimation seconds after pod scheduled.
+	AnnotationCustomForceEstimationSecondsAfterPodScheduled = SchedulingDomainPrefix + "/force-load-estimation-seconds-after-pod-scheduled"
```

Comment:
Apart from the estimating window size, perhaps we can allow the pod to customize its scaling factor, WDYT?